doc/releases/changelog-dev.md

```diff
@@ -242,6 +242,8 @@
 * A note about the eigenspectrum of second-quantized Hamiltonians added to `qml.eigvals`.
   [(#5095)](https://github.com/PennyLaneAI/pennylane/pull/5095)
 
+* Added a reference to the paper that provides the image of the `qml.QAOAEmbedding` template. [(#5130)](https://github.com/PennyLaneAI/pennylane/pull/5130)
+
```

Comment:
Feel free to optionally add your name to the list of contributors at the bottom.